aries_cloudagent/config/argparse.py

```diff
@@ -760,6 +767,32 @@ def get_settings(self, args: Namespace):
         return settings
 
 
+@group(CAT_START)
+class MediationGroup(ArgumentGroup):
+    """Mediation settings."""
+
+    GROUP_NAME = "Mediation"
+
+    def add_arguments(self, parser: ArgumentParser):
+        """Add mediation command line arguments to the parser."""
+        parser.add_argument(
+            "--open-mediation",
+            metavar="<label>",
+            help="Enables didcomm mediation. After establishing a connection, if enabled, \
+                an agent may request message mediation, which will allow the mediator to \
+                forward messages on behalf of the recipient. See aries-rfc:0211."
```

Comment:
The open mediation flag indicates that we are willing to automatically grant mediation requests and begin mediating messages for the connection rather than enabling mediation altogether. I think we may have talked past each other a bit on that point so hopefully that clears it up.

Reply:
check if the next PR clears up the confusion. I am open to any direction.